### PR TITLE
updated voxceleb v1 preparation scripts

### PR DESCRIPTION
**Pull request by jyhnnhyj**
@david-ryan-snyder as discussed, please have a look

### CODE REVIEW

File: egs/dihard_2018/v1/run.sh [hunk 1]
```diff
@@ -28,9 +28,9 @@ stage=0
 if [ $stage -le 0 ]; then
   local/make_voxceleb2.pl $voxceleb2_root dev data/voxceleb2_train
   local/make_voxceleb2.pl $voxceleb2_root test data/voxceleb2_test
-  # This script creates data/voxceleb1_test and data/voxceleb1_train.
   # Our evaluation set is the test portion of VoxCeleb1.
-  local/make_voxceleb1.pl $voxceleb1_root data
```
Comment (david-ryan-snyder) [inline egs/dihard_2018/v1/run.sh]: Could you add a comment saying something like this:

```
# Now prepare the VoxCeleb1 train and test data.  If you downloaded the corpus soon
# after it was first released, you may need to use an older version of the script, which
# can be invoked as follows:
# local/make_voxceleb1.pl $voxceleb1_root data
```

File: egs/dihard_2018/v1/run.sh [hunk 2]
```diff
@@ -28,9 +28,9 @@ stage=0
 if [ $stage -le 0 ]; then
   local/make_voxceleb2.pl $voxceleb2_root dev data/voxceleb2_train
   local/make_voxceleb2.pl $voxceleb2_root test data/voxceleb2_test
-  # This script creates data/voxceleb1_test and data/voxceleb1_train.
   # Our evaluation set is the test portion of VoxCeleb1.
```
Comment (david-ryan-snyder) [inline egs/dihard_2018/v1/run.sh]: Can you remove the comment "Our evaluation set is the test portion of VoxCeleb1." It seems to be referring to a different recipe.

File: egs/dihard_2018/v2/run.sh [hunk 3]
```diff
@@ -27,9 +27,9 @@ stage=0
 if [ $stage -le 0 ]; then
   local/make_voxceleb2.pl $voxceleb2_root dev data/voxceleb2_train
   local/make_voxceleb2.pl $voxceleb2_root test data/voxceleb2_test
-  # This script creates data/voxceleb1_test and data/voxceleb1_train.
   # Our evaluation set is the test portion of VoxCeleb1.
-  local/make_voxceleb1.pl $voxceleb1_root data
```
Comment (david-ryan-snyder) [inline egs/dihard_2018/v2/run.sh]: Please leave some comment, like the one I provided earlier, saying that users may need to use an older version of the script.

File: egs/dihard_2018/v2/run.sh [hunk 4]
```diff
@@ -27,9 +27,9 @@ stage=0
 if [ $stage -le 0 ]; then
   local/make_voxceleb2.pl $voxceleb2_root dev data/voxceleb2_train
   local/make_voxceleb2.pl $voxceleb2_root test data/voxceleb2_test
-  # This script creates data/voxceleb1_test and data/voxceleb1_train.
   # Our evaluation set is the test portion of VoxCeleb1.
```
Comment (david-ryan-snyder) [inline egs/dihard_2018/v2/run.sh]: Also, please remove this comment that seems to refer to a different recipe.

**Comment by david-ryan-snyder** [comment]
Thanks a lot @jyhnnhyj. It would be great if you could add a couple of comments to the run.sh files saying that there's an older version of the dataprep script which should be used if you have an older version of the corpus. After that, it should be able to be merged.

**Comment by jyhnnhyj** [comment]
> Thanks a lot @jyhnnhyj. It would be great if you could add a couple of comments to the run.sh files saying that there's an older version of the dataprep script which should be used if you have an older version of the corpus. After that, it should be able to be merged.

sure, all done

**Comment by david-ryan-snyder** [comment]
Thanks. @danpovey, can you merge this when you have a chance?

**Pull request merged**